src/index.html

```diff
@@ -1,15 +1,100 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST"
+    onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input type="text" id="surname"
+        name="surname" required
+        inlength="3">
+        <br>
+        <label for="name">Name:</label>
+        <input type="text" id="name"
+        name="name"
+        required maxlength="20">
+        <br>
+        <label for="age">How old are You?</label>
+        <input type="number" id="age"
+        name="age" value="12"
+        required min="0"
+        max="100">
+        <br>
+        <label for="date">Full date of birth:</label>
+        <input type="date" name="date"
+        id="date" min="0000-00-00">
+        <br>
+        <label for="accept">I accept the term of the agreement</label>
+        <input type="checkbox" name="accept"
+        id="accept">
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="email">E-mail:</label>
+        <input type="email" placeholder="email@example.com"
+        name="email" id="email"
+        multiple>
+        <br>
+        <label for="password">Password:</label>
+        <input type="password" name="password"
+        id="password">
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <p>Do you love cats?</p>
+        <input type="radio" name="radioButton"
+        id="radioButtonNo" value="yes">
+        <label for="radioButtonNo">Yes</label>
```

Comment:
```suggestion
        <label for="radioButtonNo">
          Yes
          <input
            type="radio"
            name="radioButton"
            id="radioButtonNo" 
            value="yes"
          >
        </label>
```
отредактируй все в соотвествии с этим примером